google-cloud-firestore/google-cloud-firestore.gemspec

```diff
@@ -19,10 +19,7 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = ">= 2.4"
 
-  gem.add_dependency "google-cloud-core", "~> 1.2"
-  gem.add_dependency "google-gax", "~> 1.8"
-  gem.add_dependency "googleapis-common-protos", ">= 1.3.9", "< 2.0"
-  gem.add_dependency "googleapis-common-protos-types", ">= 1.0.4", "< 2.0"
+  gem.add_dependency "google-cloud-core", "~> 1.5"
```

Comment:
I'm expecting `google-cloud-firestore-v1` and `google-cloud-firestore-admin-v1` here.

Reply:
Should it just be `google-cloud-firestore-v1` for now? `google-cloud-firestore-admin-v1` isn't currently used in this gem.